nomadhook/units/UEL0001/UEL0001_script.lua

```diff
@@ -4,19 +4,14 @@ UEL0001 = Class(oldUEL0001) {
 	CreateEnhancement = function(self, enh)
 		oldUEL0001.CreateEnhancement(self, enh)
 
-		if enh =='TacticalMissile' then
-			LOG('Tac Missile Attached')
-			self:GetBlueprint().SpecialAbilities.LaunchTacMissile.NoAutoEnable = false
-			self:RegisterSpecialAbilities()
-        elseif enh == 'TacticalNukeMissile' then
-			LOG('Billy Attached')
-			self:RegisterSpecialAbilities()
-		elseif enh == 'TacticalMissileRemove' or enh == 'TacticalNukeMissileRemove' then
-			LOG('Tac or Billy Removed')
-			self:GetBlueprint().SpecialAbilities.LaunchTacMissile.NoAutoEnable = true
-			self:UnregisterSpecialAbilities()
+	if enh =='TacticalMissile' then
+		self:GetBlueprint().SpecialAbilities.LaunchTacMissile.NoAutoEnable = false
+		self:RegisterSpecialAbilities()
+	elseif enh == 'TacticalMissileRemove' then
+		self:GetBlueprint().SpecialAbilities.LaunchTacMissile.NoAutoEnable = true
+		self:UnregisterSpecialAbilities()

```

Comment:
you should disable the button as well when you upgrade the billy nuke (on finishing the upgrade, in case it gets canceled, but this is done in this function anyway.)
Plz add 
```
elseif enh = 'TacticalNukeMissile' then
    self:GetBlueprint().SpecialAbilities.LaunchTacMissile.NoAutoEnable = true
    self:UnregisterSpecialAbilities()
```